pkgs/applications/virtualization/virt-manager/default.nix

```diff
@@ -53,8 +54,14 @@ python3Packages.buildPythonApplication rec {
     gappsWrapperArgs+=(--prefix PATH : "${makeBinPath [ cpio e2fsprogs file findutils gzip ]}")
   '';
 
-  # Failed tests
-  doCheck = false;
+  checkInputs = [ pytest cpio cdrtools ];
+  checkPhase = ''
+    export HOME="."
+    pytest --ignore tests/test_cli.py \
+           --ignore tests/test_disk.py \
+           --ignore tests/test_checkprops.py
+    $out/bin/virt-manager --version | grep ${version} > /dev/null
```

Comment:
For this part, you can then use `postCheck` or, better, [`passthru.tests`](https://github.com/NixOS/nixpkgs/pull/119731/files).

Reply:
Done.